### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Create store.
- Create "simple" action.
- Create reducer for book list.
- Add plugin reactotron-redux.

## Screenshots
Captura de Pantalla 2021-05-21 a la(s) 2 23 32 p  m

## Trello Card

https://trello.com/c/QDiMtNGG/25-n-91-redux-i-actions-reducers-store

### CODE REVIEW

File: wbooks/src/redux/store.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { createStore } from 'redux';
+import Reactotron from '@config/reactotronConfig';
+import bookReducer from '@redux/book/reducer';
+
+export default createStore(bookReducer, Reactotron.createEnhancer());
```
Comment (guidoprinc) [inline wbooks/src/redux/store.ts]: bookReducer is one of many. The store should receive an array of reducers even if you only have one

File: wbooks/src/app/screens/Library/index.tsx [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+}
+
 const Library = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
+  const books = useSelector((state: RootState) => state.books);
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Library/index.tsx:21]: when you modify `store.ts` the books will be in `state.books.books`

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/Library/index.tsx (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Library/index.tsx:21]: Ok, fixed

File: wbooks/src/redux/store.ts (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/redux/store.ts]: in the new commit, I put a combine reducer

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**